### DISCUSSION TITLE
how to configure a cluster with deployment mode?

### DISCUSSION THREAD

**Discussion by Cheol-Soon-Choi** [apache/apisix#8515] [Q&A]
trying to configure apisix cluster for high availability.
I wonder if the deployment mode is related to apisix cluster configuration.
If relevant, which of the three methods below is appropriate?
![스크린샷 2022-12-13 오후 8 27 17](https://user-images.githubusercontent.com/79562672/207306204-9067ae66-c957-4a7f-9ab2-54ec44dfb7e4.png)

I'm a newbie.. so I'd appreciate it if you could explain it in an easy way.

added questions)
In the traditional way, two apisix instances were connected to etcd.
Is it normal that only one node is shown in the dashboard system-info(APISIX Nodes)??

**Comment by tzssangglass** [comment]
> If relevant, which of the three methods below is appropriate?

form your image, Decoupled2 is better.

> In the traditional way, two apisix instances were connected to etcd.
> Is it normal that only one node is shown in the dashboard system-info(APISIX Nodes)??

it's abnormal

**Reply by mocuishlelin** [reply]
> Is it normal that only one node is shown in the dashboard system-info(APISIX Nodes)??

Have you completely copied the apisix directory so that the id of apisix in etcd will be the same and you will only see the information of one node
